un_sdg/output/standardized_entity_names.csv

```diff
@@ -0,0 +1,342 @@
+country_code,Our World In Data Name

```

Comment:
Suggestion: move this file to `un_sdg/config/standardized_entity_names.csv`.

I changed [README.md](https://github.com/owid/importers#readme) a few weeks ago proposing that we use a `config/` folder for files that are constructed in some manual way (like `standardized_entity_names.csv`). Not sure this is necessarily a good solution in the longer term, but something to try out in the short term.